docs/admin/performance.rst

```diff
@@ -83,6 +83,18 @@ It is recommended that you do this on your development platform if you are writi
 This cache is automatically flushed when a plugin is enabled, disabled or reordered,
 or when upgrade.php is executed.
 
+For best performance, you can also create a symlink from ``/cache/`` in your www
+root dir to the ``/views_simplecache/`` directory in the data directory you
+configured when you installed Elgg:
+
+.. code:: shell
+
+    cd /path/to/wwwroot/
+    ln -s cache /path/to/dataroot/views_simplecache/
+
+If your webserver supports following symlinks, this will serve files straight off
```

Comment:
Yea, the cache is lazily filled up. I think it would be cool to provide a command eventually to generate all the static views up front, but one thing at a time.